example/package.json

```diff
@@ -10,13 +10,14 @@
   },
   "dependencies": {
     "react": "18.2.0",
-    "react-native": "0.71.3"
+    "react-native": "0.71.3",
```

Comment:
can you go latest version of `0.71` not really sure but some of these old patch version numbers have compatibility issues with latest xcode